src/nxdoc/nuxeo-server/authentication-and-user-management/using-oauth2.md

```diff
@@ -86,6 +86,31 @@ OAuth 2 is natively supported by the Nuxeo Platform, which means there is no bun
 Nuxeo allows you to register an OAuth 2 client by specifying an arbitrary name, a client ID, possibly a client secret and a list of redirect URIs.
 There are two ways of registering a client.
 
+### Through the Admin Center
+{{{multiexcerpt 'JSF-UI-required' page='generic-multi-excerpts'}}}
+
+#### Using Web UI
```

Comment:
Shouldn't it be outside of _Through the Admin Center_ section as it states _JSF UI Required_?

Reply:
Yes, you're right, I'll make sure to find a way to display correctly the info. Thanks!